fastlane_core/lib/fastlane_core/configuration/configuration.rb

```diff
@@ -219,7 +219,8 @@ def fetch(key, ask: true)
       value = if @values.key?(key) && !@values[key].nil?
                 @values[key]
               elsif option.env_name && !ENV[option.env_name].nil?
-                ENV[option.env_name].dup
+                # Convert and verify! before using it (https://github.com/fastlane/fastlane/issues/14449)
```

Comment:
Does the conversion actually get used in the end or is it only for the `verify!` call? Then this might be formulated confusingly.

Reply:
It is only for the `verify!` call due to part of the validation being the boolean check within the method.  [This test](https://github.com/fastlane/fastlane/blob/master/fastlane/spec/actions_specs/say_spec.rb#L56) fails when verifying before.  

I tried moving the verify around in a variety of ways, most of which resulted in a number of failed tests. This one seemed to preserve the most identical functionality while addressing the verify issue directly.

Reply:
```suggestion
                # verify! before using (see https://github.com/fastlane/fastlane/issues/14449)
```